modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -151,13 +155,50 @@ def update_status!(status:, code: nil, detail: nil)
                                       statusable_id: id
                                     })
 
+      email_handler = Events::Handler.new(event_type: :nod_received, opts: {
+                                            email_identifier: email_identifier,
+                                            first_name: veteran_first_name,
+                                            date_submitted: veterans_local_time.iso8601,
+                                            guid: id
+                                          })
+
       update!(status: status, code: code, detail: detail)
 
       handler.handle!
+      email_handler.handle! if status == 'submitted' && email_identifier.present?
     end
 
     private
 
+    def mpi_veteran
+      AppealsApi::Veteran.new(
+        ssn: ssn,
+        first_name: veteran_first_name,
+        last_name: veteran_last_name,
+        birth_date: birth_date.iso8601
+      )
+    end
+
+    def email_identifier
+      return { id_type: 'email', id_value: email } if email.present?
```

Comment:
The `email` method will need to work for both NOD v1 and v2 since we changed the key from `emailAddressText` to `email`.

I'll add a ticket to circle back around to make sure v1 and v2 work.